api/cpp/cmake/SlintMacro.cmake

```diff
@@ -9,6 +9,7 @@ set_property(CACHE DEFAULT_SLINT_EMBED_RESOURCES PROPERTY STRINGS
 ## This requires CMake 3.23 and does not work in 3.26 AFAICT.
 # define_property(TARGET PROPERTY SLINT_EMBED_RESOURCES
 #     INITIALIZE_FROM_VARIABLE DEFAULT_SLINT_EMBED_RESOURCES)
+set(DEFAULT_SLINT_SCALE_FACTOR "" CACHE STRING "A constant scale factor to apply. Use only in MCU environments")
```

Comment:
why is this `DEFAULT_`?  shouldn't all these var start by `SLINT_` so it should be `SLINT_DEFAULT_SCALE_FACTOR` or just `SLINT_SCALE_FACTOR` or perhaps `SLINT_CONST_SCALE_FACTOR`

Reply:
It's the same as with the resource embedding: This cache property comes from the application, not Slint. We can discuss with @hunger more, but if you're unsure I can also remove this fallback for now.